src/main/resources/2.7.x.conf

```
@@ -98,7 +99,9 @@ templatecontrol {
         conversions = {
 
           "def scalaVersion" = """def scalaVersion = System.getProperty("scala.binary.version", /* default = */ "2.12")"""
-          "def playVersion" = """def playVersion = "2.7.0-RC8""""
+          "def playVersion" = """def playVersion = "2.7.0-RC9""""
+
+          """play "org.webjars:webjars-play""" = """    play "org.webjars:webjars-play_$scalaVersion:2.7.0-RC9"""
```

Should this be:
```suggestion
          """play "org.webjars:webjars-play""" = """    play "org.webjars:webjars-play_\$scalaVersion:2.7.0-RC9"""
```
?

---

no, for the same reasons mentioned earlier.

We should only escape for Giter8 templates.